Dockerfile for automated docker deployment

kaifabian opened the pull request:
This PR contains a two stage Dockerfile that can be used on Dockerhub to automatically build a om-parser-stw-potsdam-v2 image for deployment via Docker (or alternatively, a local deployment using "docker build"). It is based on Alpine Linux with Python 2.7 (and uwsgi as frontend service).

The first stage runs the provided tests (via "make test"). If those succeed, the actual image is built.

f4lco commented on Dockerfile:
```diff
@@ -0,0 +1,57 @@
+RUN chown -R flaskd:flaskd .
+
+USER flaskd
+ENV PYTHONPATH="/opt/om-parser-stw-potsdam-v2:${PYTHONPATH}"
```
LGTM, works like a charm. In order to fully understand, just two minors:
* Why tinker with `PYTHONPATH`? I does not appear relevant to me, I suppose pipenv should take care of that.
* Maybe a (env) variable for the app path would be nice, instead of spelling it out at least five times.

kaifabian commented:
Both your comments should be taken care (commits c1bee2d and d6973fc). Additionaly, I parameterized the listen port (74077a7), removed the tests and Makefile in the final image, since they are not required there (2fd1889) and added a health check that can be used in HA deployments (01d4067).
Better? :)
